pinot-common/src/main/java/org/apache/pinot/filesystem/PinotFSFactory.java

```diff
@@ -43,6 +43,18 @@ private PinotFSFactory() {
 
   }
 
+  public static void register(String scheme, String fsClassName, Configuration configuration) {
+    try {
+      LOGGER.info("Initializing PinotFS for scheme {}, classname {}", scheme, fsClassName);
+      PinotFS pinotFS = (PinotFS) Class.forName(fsClassName).newInstance();
+      pinotFS.init(configuration);
+      _fileSystemMap.put(scheme, pinotFS);
+    } catch (InstantiationException | IllegalAccessException | ClassNotFoundException e) {
+      LOGGER.error("Could not instantiate file system for class {}", fsClassName, e);
```

Comment:
Please include schema and fsclassname in the log

Reply:
will do